cmd/nerdctl/compose_run_linux_test.go

```diff
@@ -432,12 +432,13 @@ func TestComposePushAndPullWithCosignVerify(t *testing.T) {
 	keyPair := newCosignKeyPair(t, "cosign-key-pair")
 	defer keyPair.cleanup()
 
-	reg := testregistry.NewPlainHTTP(base, 5000)
-	defer reg.Cleanup()
+	reg := testregistry.NewWithNoAuth(base, 0, false)
```

Comment:
Why renamed?

Reply:
Methods have been simplified / renamed:
- there is no longer separate helpers for http vs. https (you pass a tls bool instead) - so, the previous name (NewPlainHTTP, that does not specific what type of auth it is using)
- all helpers names now follow the same pattern:
  * NewWithTokenAuth
  * NewWithBasicAuth
  * NewWithNoAuth

Let me know if you don't like it and I can revert this one if you wish.